src/helpers/types/models.ts

```diff
@@ -100,8 +142,17 @@ export interface BeneficiaryAttributes {
 }
 
 export interface ManagerAttributes {
-    user: string;
+    id: number;
+    address: string;
     communityId: string;
+    active: boolean;
+
+    // timestamps
+    createdAt: Date;
+    updatedAt: Date;
+
+    user?: User;
```

Comment:
Is really possible to have a manager that doesn't have a user or community associate with it?

Reply:
No, they always have a user and community. But this is a generic ManagerAttribute return from the API/DB. It might or might not bring user and/or community information.